inspirehep/modules/workflows/static/js/inspire_workflows_ui/holdingpen/holdingpen.filters.js

```diff
@@ -78,7 +78,10 @@
         }
       }
 
-      return categories;
+      var uniqueCategories = categories.filter(function (value, index, self) {
+        return self.indexOf(value) === index;
+      });
```

Comment:
Well, not that it matters, since we're dealing with something like 2/3 elements anyway...